tests/sentry/integrations/slack/test_integration.py

```diff
@@ -3,7 +3,7 @@
 import responses
 
 from sentry.integrations.slack import SlackIntegration, SlackIntegrationProvider
-from sentry.integrations.slack.utils import SLACK_GET_USERS_PAGE_SIZE
+from sentry.integrations.slack.utils.users import SLACK_GET_USERS_PAGE_SIZE
```

Comment:
@mgaeta should we export these constants from the `__init__.py`?

Reply:
So I thought about this a lot. I think that if we had page sizes shared by multiple endpoints it would make sense to centralize them. In this case `SLACK_GET_USERS_PAGE_SIZE` is only relevant to this one file and the file's tests.

Reply:
I'll just put my 2 cents in, I think if in the future we end up using the constants elsewhere, this rule you've described probably won't get upheld, they'll still sit in utils/users.py. `SLACK_RATE_LIMITED_MESSAGE` is already in the__init__.py file and I personally would like if all the other constants were as well (e.g. this one, `CHANNEL_PREFIX`, `MEMBER_PREFIX`, etc.)

Reply:
The other consideration I'm using it trying to limit the interface of the module. This list in `__init__.py` only includes functions and variables referenced in other `/src/` code. 

```python
__all__ = (
    "get_slack_data_by_user",
    "get_users",
    "set_signing_secret",
    "build_notification_footer",
    "check_signing_secret",
    "get_channel_id",
    "get_channel_id_with_timeout",
    "logger",
    "send_confirmation",
    "send_incident_alert_notification",
    "send_slack_response",
    "strip_channel_name",
    "validate_channel_id",
    "SLACK_RATE_LIMITED_MESSAGE",
)
```
To me, code in `/tests/` is special and can do all sorts of hacky things in imports.